data/boarding missions.txt

```diff
@@ -37,3 +37,29 @@ mission "Assisting Hai"
 			`Knowing you risked your life to save the <origin>, the captain gives you <payment>.`
 				decline
 	destination Hai-home
+
+mission "Navy emergency"
+	assisting
+	repeat
+	deadline
+	passengers 1 3
+	description "Deliver <fare> to <destination> by <date>; the military hospital is waiting for you."
+	to offer
+		"reputation: Navy (Oathkeeper)" > 100
+		random < 15
+	source
+		government "Navy (Oathkeeper)"
+		system "Arneb" "Hatysa" "Alnilam" "Unagi" "Almaaz" "Mintaka" "Gorvi" "Tortor" "Fah Root" "Rigel" "Hassaleh" "Danoa" "Rajak" "Sumar" "Ultima Thule" "Cardax" "Volax" "Moktar"
```

Comment:
Oathkeepers only appear in Canopus, Alheka, Mirzam, Mebsuta, Betelgeuse, Rigel, Alnitak, Saiph, Mintaka,  Almaaz, Gorvi, and Tortor.